### ISSUE TITLE
Does fortigate exporter image has /bin/sh "shell" available ?

### ISSUE THREAD

**Issue by faizansyed66** [bluecmd/fortigate_exporter#261]
I would like to ask if fortigate exporter image has shell available ? I want to run some shell script before starting a container.
As I am running fortigate exporter in cloud foundry so I cannot mount fortigate-key.yml file directly as volume because cloud foundry does not support file mounting.
So I would like to run some shell script as ENTRYPOINT in my docker file to read API token and copy it to /config/fortigate-key.yaml path inside the container.
Can you please advice ? 
@bluecmd @neilschelly

**Comment by bluecmd** [comment]
I'd recommend building your own custom docker image for this purpose, it's not in scope for the project to carry extra programs for convenience - sorry!
